### PR TITLE
Add 3 new sites.

### PR DESCRIPTION
**Pull request by Firstbober**
This pull request adds entries for the following services:
- VSTBuzz
- RapidAPI
- Sketchfab

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -6003,6 +6003,16 @@
         ]
     },
 
+    {
+        "name": "RapidAPI",
+        "url": "https://rapidapi.com/",
```
Comment (Nightfirecat) [inline _data/sites.json]: What's the link/email to contact support?

Comment (Firstbober) [reply _data/sites.json]: I added email to support and link to contact page.

File: _data/sites.json [hunk 2]
```diff
@@ -7487,6 +7507,16 @@
         ]
     },
 
+    {
+        "name": "VSTBuzz",
+        "url": "https://vstbuzz.com",
```
[the remark this answers is not in the document]
Comment (Firstbober) [reply _data/sites.json]: Here, I changed link to page with contact form.

**Comment by Firstbober** [comment]
Corrected.

File: _data/sites.json [hunk 3]
```diff
@@ -6003,6 +6003,17 @@
...
+        "name": "RapidAPI",
+        "url": "https://rapidapi.com/contact-us",
+        "difficulty": "hard",
+        "notes": "Account can be only deleted with sending request to support",
+        "email": "support@rapidapi.com",
+        "domains": [
+            "rapidapi.com"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json]: This needs to be moved upwards to keep the alphanumeric ordering and pass on the Travis CI
Put it between "Rainforest QA" and "RateYourMusic"

File: _data/sites.json [hunk 4]
```diff
@@ -6381,6 +6392,16 @@
...
+    {
+        "name": "Sketchfab",
+        "url": "https://sketchfab.com/settings/account",
+        "difficulty": "easy",
+        "notes": "You have to click on the button Delete your account, select the consent to delete all content and confirm.",
+        "domains": [
+            "sketchfab.com"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json]: This has to be between "Simplenote" and "Skillshare", to preserve the alphanumeric order

**Comment by Firstbober** [comment]
Changed order.

File: _data/sites.json [hunk 5]
```diff
@@ -5973,6 +5973,17 @@
         ]
     },
 
+    {
+        "name": "RapidAPI",
+        "url": "https://rapidapi.com/contact-us",
+        "difficulty": "hard",
+        "notes": "Account can be only deleted with sending request to support",
```
Comment (tupaschoal) [inline _data/sites.json]:
```suggestion
        "notes": "Account can only be deleted by sending a request to their support team",
```

File: _data/sites.json [hunk 6]
```diff
@@ -6346,6 +6357,16 @@
         ]
     },
 
+    {
+        "name": "Sketchfab",
+        "url": "https://sketchfab.com/settings/account",
+        "difficulty": "easy",
+        "notes": "You have to click on the button Delete your account, select the consent to delete all content and confirm.",
```
Comment (tupaschoal) [inline _data/sites.json]:
```suggestion
        "notes": "You have to click on the button 'Delete your account', select the 'consent to delete all content' and confirm.",
```

File: _data/sites.json [hunk 7]
```diff
@@ -7487,6 +7508,16 @@
         ]
     },
 
+    {
+        "name": "VSTBuzz",
+        "url": "https://vstbuzz.com/contact/",
+        "difficulty": "hard",
+        "notes": "Account can be only deleted with sending request to support",
```
Comment (tupaschoal) [inline _data/sites.json]:
```suggestion
        "notes": "Account can only be deleted by sending request to their support team",
```

File: _data/sites.json (hunk 5 above)
Comment (Firstbober) [reply _data/sites.json]: Looks better.

File: _data/sites.json (hunk 6 above)
Comment (Firstbober) [reply _data/sites.json]: This too.

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**